lib/processor/deps.edn

```diff
@@ -9,29 +9,29 @@
 
         garden/garden {:mvn/version "1.3.10"}
         ;girouette/girouette {:local/root "../girouette"}
-        girouette/girouette {:mvn/version "0.0.2"}}
+        girouette/girouette {:mvn/version "0.0.3"}}
 
  :aliases {:test {:extra-paths ["test"]
                   :extra-deps  {lambdaisland/kaocha           {:mvn/version "1.0.641"}
                                 lambdaisland/kaocha-cljs      {:mvn/version "0.0-71"}
                                 lambdaisland/kaocha-junit-xml {:mvn/version "0.0.76"}
                                 org.clojure/test.check        {:mvn/version "1.1.0"}}}
 
-           ; clojure -M:outdated --write
+                                        ; clojure -M:outdated --write
            :outdated {:replace-deps {olical/depot {:mvn/version "2.1.0"}}
-                      :main-opts ["-m" "depot.outdated.main"]}
+                      :main-opts    ["-m" "depot.outdated.main"]}
 
            :reveal {:extra-deps {vlaaad/reveal {:mvn/version "1.3.194"}}
                     :ns-default vlaaad.reveal
-                    :exec-fn repl}
+                    :exec-fn    repl}
 
            :depstar {:replace-deps {seancorfield/depstar {:mvn/version "2.0.188"}}
-                     :exec-fn hf.depstar/jar
-                     :exec-args {:sync-pom true
-                                 :group-id "girouette"
-                                 :artifact-id "processor"
-                                 :version "0.0.2"
-                                 :jar "processor.jar"}}}}
+                     :exec-fn      hf.depstar/jar
+                     :exec-args    {:sync-pom    true
+                                    :group-id    "girouette"
+                                    :artifact-id "processor"
+                                    :version     "0.0.3"
```

Comment:
This is the version for the release tool. I will change it when it is time to make a release of the processor.